lib/storage/s3.js

```diff
@@ -94,6 +94,9 @@ module.exports = function() {
       client = new AWS.S3(options);
       if (options.contentTypes) {
         contentTypes = options.contentTypes;
+        if (!contentTypes.svg) {
+          contentTypes.svg = 'image/svg+xml';
```

Comment:
We don't force in values for other content types when the user explicitly passes the types they want, so I'm not sure why we'd want to do that for SVG.